addons/GME/Scripts/Game/GME/AI/Components/SCR_AICombatComponent.c

```diff
@@ -13,12 +13,16 @@ modded class SCR_AICombatComponent : ScriptComponent
 		if (!outCurrentTarget)
 			return;
 		
-		bool visible = GME_VisibilityHelper.GetVisibility(outCurrentTarget.GetTargetEntity());
-		if (visible)
+		IEntity currentEntity = outCurrentTarget.GetTargetEntity();
+		if (!currentEntity)
```

Comment:
Is it possible to have a  valid "outCurrentTarget" without a valid Entity? I assumed a valid target would always be an entity of some kind? 

Although I am all for covering possible edge-cases and undefined behaviour by adding checks ;-)

Reply:
Null checks are pretty much always done for `BaseTarget::GetTargetEntity` in the vanilla codebase.